cirq-core/cirq/qis/__init__.py

```diff
@@ -60,3 +60,4 @@
     average_error,
     decoherence_pauli_error,
 )
+from cirq.qis.process_renyi_entropy_from_bitstrings import process_entropy_from_bitstrings
```

Comment:
`cirq.qis` now contains both `process_entropy_from_bitstrings` and `process_renyi_entropy_from_bitstrings`, where `process_entropy_from_bitstrings` is a function within `process_renyi_entropy_from_bitstrings`. This is confusing.

Reply:
Renamed file to entropy.py. Leaving the entropy work in this PR, which can later be optimized by @Gaurav